data-raw/adlb_grading/atoxgr_sources.R

```diff
@@ -45,3 +45,40 @@ atoxgr_criteria_daids_uscv <- atoxgr_criteria %>%
   dplyr::mutate(GRADE_CRITERIA_CODE = gsub("[\r\n]", " ", GRADE_CRITERIA_CODE))
 
 save(atoxgr_criteria_daids_uscv, file = "data/atoxgr_criteria_daids_uscv.rda")
+
+# Read in JSON file - testing
+json_file_path <- "data_raw/adlb_grading/adlb_grading_spec_ncictcaev5.json"
+
+atoxgr_criteria_ctcv5_uscv <- fromJSON(json_file_path) %>%
+  filter(!is.na(GRADE_NA_CODE)) %>%
```

Comment:
Can `GRADE_NA_CODE` be `NA`?

Reply:
Not the way I have set up, always define at least `is.na(AVAL)` as first line in case statement.

Reply:
Yeah, this sounds find to me, just to clairfy we would check the final case statement can be parsed in the `atoxgr_sources` script?